eva-accession-pipeline/src/main/resources/application.properties

```diff
@@ -20,4 +15,13 @@ parameters.vcfAggregation=
 parameters.fasta=
 
 parameters.chunkSize=
-parameters.forceRestart=
\ No newline at end of file
+parameters.forceRestart=
+
+
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=
+spring.datasource.username=
+spring.datasource.password=
+
+# Only to set up the database!
+# spring.jpa.generate-ddl=true
```

Comment:
To be honest I prefer the granularity `spring.jpa.hibernate.ddl-auto` provides, because we can set it to "validate" so it checks the schema every time the application is started.